test/built-ins/Object/defineProperty/coerced-P-grow.js

```diff
@@ -11,16 +11,14 @@ includes: [compareArray.js, resizableArrayBufferUtils.js]
 features: [resizable-arraybuffer]
 ---*/
 
-function ObjectDefinePropertyHelper(ta, index, value) {
+function MayNeedBigInt(ta, value) {
   if (ta instanceof BigInt64Array || ta instanceof BigUint64Array) {
```

Comment:
Future work: `MayNeedBigInt` could go into the helper file

Reply:
I'll make a followup PR with this change, since this is defined across many files.